Library/Tracking/Koala.swift

```diff
@@ -1963,53 +1964,46 @@ public final class Koala {
     )
   }
 
-  private func defaultProperties() -> [String: Any] {
+  // MARK: - Session Properties
+
+  private func sessionProperties(_ prefix: String = "session_") -> [String: Any] {
     var props: [String: Any] = [:]
 
     let enabledFeatureFlags = self.config?.features
       .filter { key, value in key.starts(with: "ios_") && value }
       .keys
       .sorted()
 
-    props["manufacturer"] = "Apple"
-    props["app_version"] = self.bundle.infoDictionary?["CFBundleVersion"]
-    props["app_release"] = self.bundle.infoDictionary?["CFBundleShortVersionString"]
-    props["current_variants"] = self.config?.abExperimentsArray.sorted() ?? []
-    props["enabled_feature_flags"] = enabledFeatureFlags ?? []
-    props["model"] = Koala.deviceModel
-    props["distinct_id"] = self.distinctId
+    props["apple_pay_capable"] = PKPaymentAuthorizationViewController.applePayCapable()
+    props["apple_pay_device"] = PKPaymentAuthorizationViewController.applePayDevice()
+    props["cellular_connection"] = CTTelephonyNetworkInfo().serviceCurrentRadioAccessTechnology
+    props["client_type"] = "native"
+    props["current_variants"] = self.config?.abExperimentsArray.sorted()
+
+    props["device"] = self.device.name
     props["device_fingerprint"] = self.distinctId
+    props["device_format"] = self.deviceFormat
+    props["device_manufacturer"] = "Apple"
+    props["device_model"] = Koala.deviceModel
+    props["device_orientation"] = self.deviceOrientation
+    props["distinct_id"] = self.distinctId
```

Comment:
As discussed on Slack this, `device_fingerprint` and `iphone_uuid` all appear to be the same value.

Reply:
I'll bring it up with insights and remove one property in a follow-up PR if they're ok with it.

Reply:
Not remove two props?

Reply:
Sorry yes. Consolidate to one prop.